### DISCUSSION TITLE
Shortcuts: Are they right, and do we have too many defaults?

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#44] [General]
'Save translation' and 'Save translation as' are less likely to be used in comparison to common functions such as 'Load subtitle', 'Save', and 'Close', so they should probably not have any; if desired, a user can set them (like some others which do not have any).

I'd like some input from designers on this:
- Ensuring current ones comply with standards
- Remove those that aren't commonly used

**Comment by GitGianluc** [comment]
The descriptions of the shortcuts are rather odd and unusual on macOS: “Meta” is not the name I would expect to refer to ⌘ / Cmd, which is displayed correctly under each menu instead (File, Edit, Find, etc). Additionally, it is possible to assign the same key combination to two different commands without any error message being displayed (and this is supposed to be a bug).

**Reply by GitGianluc** [reply]
> “Meta” is not the name I would expect to refer to ⌘ / Cmd, which is displayed correctly under each menu instead (File, Edit, Find, etc). Additionally, it is possible to assign the same key combination to two different commands without any error message being displayed

@chenlung should we convert this discussion into an issue?

**Reply by chenlung** [reply]
I pointed out the Meta thing several weeks ago, but I can't remember @URUWorks's response.

I don't mind if it's converted if we can wait a day or two for agreement?

**Reply by URUWorks** [reply]
It is something that cannot be changed for now, Lazarus uses "Meta" as identifier on Darwin/macOS.
Additionally, I'll see that it notifies you if the shortcut is already in use.

**Reply by GitGianluc** [reply]
OK, thanks for that!
